specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -564,621 +715,1772 @@
+            }
           },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/tropical/storms/forecasts/{format}": {
+      "get": {
+        "description": "**Get Tropical Storm Forecasts**\n\n**Applies to**: Gen 1 (S0 and S1) and Gen 2 pricing tiers.\n\nGet all government-issued tropical storm forecasts. Information about the forecasted tropical storms includes, location, status, date the forecast was created, window, wind speed and wind radii.",
+        "operationId": "Weather_GetTropicalStormForecast",
+        "x-ms-examples": {
+          "Get Tropical Storm Forecasts": {
+            "$ref": "./examples/GetStormForecast.json"
+          }
+        },
+        "parameters": [
           {
-            "value": 13,
-            "name": "MostlyCloudyWithShowers",
-            "description": "Mostly Cloudy with Showers"
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
           },
           {
-            "value": 14,
-            "name": "PartlySunnyWithShowers",
-            "description": "Partly Sunny with Showers"
+            "$ref": "#/parameters/ApiVersion"
           },
           {
-            "value": 15,
-            "name": "Thunderstorms",
-            "description": "Thunderstorms"
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/JsonFormat"
           },
           {
-            "value": 16,
-            "name": "MostlyCloudyWithThunderstorms",
-            "description": "Mostly Cloudy with Thunderstorms"
+            "$ref": "#/parameters/WeatherYear"
           },
           {
-            "value": 17,
-            "name": "PartlySunnyWithThunderstorms",
-            "description": "Partly Sunny with Thunderstorms"
+            "$ref": "#/parameters/RequiredWeatherBasinId"
           },
           {
-            "value": 18,
-            "name": "Rain",
-            "description": "Rain"
+            "$ref": "#/parameters/RequiredWeatherGovernmentId"
           },
           {
-            "value": 19,
-            "name": "Flurries",
-            "description": "Flurries"
+            "$ref": "#/parameters/WeatherDataUnit"
           },
           {
-            "value": 20,
-            "name": "MostlyCloudyWithFlurries",
-            "description": "Mostly Cloudy with Flurries"
+            "$ref": "#/parameters/WeatherDetails"
           },
           {
-            "value": 21,
-            "name": "PartlySunnyWithFlurries",
-            "description": "Partly Sunny with Flurries"
+            "$ref": "#/parameters/WeatherRadiiGeometry"
           },
           {
-            "value": 22,
-            "name": "Snow",
-            "description": "Snow"
+            "name": "windowGeometry",
+            "description": "When true, window geometry data (geoJSON) is included in the response",
```

Comment:
Why is this not a $ref?  Regardless, as a bool it should be self-documenting with `x-ms-client-name`. Perhaps `include_window_geometry`.

Reply:
Added

Reply:
Maybe I missed something but I only see one windowGeometry in the file. Do you still see two definitions of windowGeometry?

Reply:
You're right. I only see the one definition.